website/static/js/statistics.js

```diff
@@ -0,0 +1,242 @@
+'use strict';
+
+require('keen-dataviz/dist/keen-dataviz.min.css');
+
+var $osf = require('js/osfHelpers');
+var keenDataviz = require('keen-dataviz');
+var keenAnalysis = require('keen-analysis');
+
+var ProjectUsageStatistics = function(){
+    var self = this;
+
+    /**
+    * Manages the connections to Keen.  Needs a project id and valid read key for that project.
+    *
+    * @property keenClient
+    * @type {keenAnalysis}
+    */
+    self.keenClient = new keenAnalysis({
+        projectId: window.contextVars.keen.public.projectId,
```

Comment:
Minor: Same here--can these be passed as arguments rather than pulling from contextVars? contextVars should only be accessed in the `-page` modules.